### PR TITLE
Use BottomSheet in place of Snackbar for longer messages

### PR DESCRIPTION
**Pull request by ashoksinghal**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [x] Refactoring
- [ ] Dependency updates

## :scroll: Description

replace some of the snack bar's which required user interactions with BasicBottomSheet to make them more readable

## :bulb: Motivation and Context

this change is required to let the user read the full message easily

https://github.com/android-password-store/Android-Password-Store/issues/1114

## :green_heart: How did you test it?
tested it via running it on mobile devices.

## :pencil: Checklist

- [x] I formatted the code with the IDE's reformat action (Ctrl + Shift + L/Cmd + Shift + L)
- [x] I reviewed submitted code
- [ ] I added a [CHANGELOG](CHANGELOG.md) entry if applicable

## :camera_flash: Screenshots / GIFs

image
image

### CODE REVIEW

File: app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt
Comment (msfjarvis) [inline app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt]: This should use `requireContext()`

File: app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt
Comment (msfjarvis) [inline app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt]: Rather than these empty setTitle calls I guess we can simply relax the checks in BasicBottomSheet.Builder to check for either rather than both.

**Review by msfjarvis — CHANGES_REQUESTED** [review]
This is a great first pass! Small changes needed but this is looking good.

File: app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt
Comment (ashoksinghal) [reply app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt]: done

File: app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt
Comment (ashoksinghal) [reply app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt]: sure @msfjarvis, it's done.

File: app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt
Comment (msfjarvis) [inline app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt]: This is also redundant now

**Review by msfjarvis — CHANGES_REQUESTED** [review]
One final change and we're ready to merge

File: app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt
Comment (ashoksinghal) [reply app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt]: Sorry, missed this. done now.

**Review by msfjarvis — APPROVED** [review]
LGTM! Will merge once I test it myself.

**Comment by ashoksinghal** [comment]
@msfjarvis thanks, Request you to please add the 'hacktoberfest-accepted' tag before merging.

**Comment by msfjarvis** [comment]
> @msfjarvis thanks, Request you to please add the 'hacktoberfest-accepted' tag before merging. 

It's only required for pull requests that aren't being merged within October. This one will be.

**Comment by ashoksinghal** [comment]
> > @msfjarvis thanks, Request you to please add the 'hacktoberfest-accepted' tag before merging.
> 
> It's only required for pull requests that aren't being merged within October. This one will be.

ok, thank you so much :)

**Pull request merged**
